base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -1664,6 +1664,16 @@ where
                     target: LOG_TARGET,
                     "Coinbase Transaction monitoring Protocol for TxId: {} completed successfully", id
                 );
+                if let Ok(tx) = self.resources.db.get_completed_transaction(id).await {
```

Comment:
So I understand that this helped diagnose a problem but you are adding in a whole DB access here for a trace log. This is something I feel is a wasted use of resources. Add them in if you have to debug an issue but leaving this here is a waste for the rest of the life of this code.

Reply:
Hi @philipr-za will remove in a next PR